docs/pages/apis/rest.fr.mdx

```diff
@@ -0,0 +1,123 @@
+# REST API
+
+Notre API RESTful fourni une façon simple d'intéragir avec la base de données de "Potter DB", en adhérant aux spécifications JSON:API et OAS. Dans cette section, nous aborderons les principaux aspects de l'utilisation de l'API REST, ce qui inclue les endpoints disponibles et la manière de formatter vos requêtes et réponses.
+
+## Endpoints disponibles
+
+Pour accéder aux données avec l'API REST vous devez faire une requête en `GET` sur l'un des endpoints suivants :
```

Comment:
```suggestion
Pour accéder aux données avec l'API REST vous devez faire une requête en `GET` sur l'un des endpoints suivants:
```

Reply:
Like for the other suggestions about spaces, I'll wait for your confirmation to apply, or not ;)